addons/mail/static/src/components/composer/composer.js

```diff
@@ -174,10 +174,22 @@ class Composer extends Component {
 
     /**
      * Post a message in the composer on related thread.
-     *
+     * Posting of the message could be aborted if it cannot be posted like if there are attachments
+     * currently uploading or if there is no text content and no attachments.
```

Comment:
Could have a separation between the 2 sentences. The 1st is the _"what"_ and the 2nd is the _"extra info"_: empty line helps this separation (note there is no _"why"_ because it doesn't feel necessary, but if wanted it, would feel natural to have it in-between these 2 paragraphs).